src/resources/websitetoolbox.js

```diff
@@ -53,33 +52,34 @@ wtbx.setting = {
             }
         }
         navigator.clipboard.writeText(copyText);
-        element.text = '';
         element.removeAttribute('data-icon');
+        if(window.screen.width >= 575){
```

Comment:
Please co-ordinate with design team, it should be fix via CSS instead of JS code.